linode/firewall/firewall_helpers_unit_test.go

```diff
@@ -0,0 +1,215 @@
+//go:build unit
+
+package firewall
```

Comment:
Should we use `firewall_test` package here (I am not sure)?

Reply:
Either way works :)